friendica_test_data.sql

```diff
@@ -1268,7 +1268,7 @@ CREATE TABLE `photo` (
������ǌc��W�������\0γ��u\"�V�F��l��F1��PAd�cs��/����',6,1,'','','',''),(19,3,7,'75859b10b87ebc39','18a7495723879503dac64b352e245508','2017-09-07 09:19:08','2017-09-07 09:19:08','','','Contact Photos','4b7db531ad1920a7cb8fb76aed46ddd9-4.jpg?ts=1504775575','image/jpeg',175,175,5493,'����\0JFIF\0\0H\0H\0\0��Created by Andi Stadler (https://snarl.de/profile/andi)\nThis work is licensed under the Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License. To view a copy of this license, visit http://creativecommons.org/licenses/by-nc-sa/4.0/.\n��\0C\0��\0\0�\0�\0��\0\0\0\0\0\0\0\0\0\0\0\0\0\n	��\0B\0\0\0\0\0\0\0	!\n1A\"Qa�q�%23�#$&5BR����Es�����\0\0\0?\0��\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\r*Ն����X\Z\'���p|L�sN���}�Y%ZR|�(�J*I&+�rJ#�yI)%l�Z�,:��i�(��ϥ�)���J�����*hχo�мI����$�Dj-� (�C\"�cI(s�_�L�:g��`w�I����9�s.w�Y���a��C\"3:I��e�~��g,�ڮ�Ϩ�6�&�t�t��/iK��xv<�v	#=�{�$�ѹ�Idу=��#�}�G���!�\0\0\0\0\0\0\0*|z�B��6�Zp�]�s\"�A�ڝ\r�E\r��{�b��Z�HI��j��>Eq�E�˒S.:t�7�Vg�5�u;��Ys=�YFZ�Rk��}����j��ʮHnl�Ifh���M6&\";M�DD�]��¿SR��c�h�B��{��t��ƤG��V�Mʽ4$�&2�\"���%���{̈}9$����489hۆ��	@�e��崭�\Z��ط<��m�(v&�Ne{�ȵE�qQMI��\Z�uO�h���D<�\0\0\0\0\0\0\0qۍGfn\Z��b��R���9~�2w���o\nstMT�x=d��>��H��l�MT���:�!M2��a��L��c��/�Ri��Cr���������T�\Z�uΥ$�F�IN��춆�q�5��S�,-B��c�^�op�}AW�8�Y)�c?��T��2Fi��դkI��$b�Փ�鐸�A��\\<���F��h�ŭ�ܿIk�)�3^�N�ٓ�wu]��k�U4�J���8�W�*�F�7�`^U:V�ң�I%�̷?��|���C{�Oh�[�:r�qğ���	����0\\���\Ze0\\�ޅ\Z��.����RvZ��NOI\n#�����Ov���-ik�F�����o�;�E�I����ձ��5�\Z�Rv!,��\\4;����MyE�3��t�a9�B��:̌�[�X�T���h_\"��r4k��*�F\\��U:��M�z�\0�u#�\0�C�\0\0\0\0\0�>�f�r���Z\\kp�qҾ.oU�u������q-S�CJK��PSǬf�2JJ��G�DYm���+>!҄�^�\n\'��/{}�W7��-K;�K���kI��(�;T�F��\r��H̖J�\0��J�sj�K��\0��m���p֫WV\'�\"IٹjTlx�;ҤO,R��Ԑǖ�edK����:�8���<)5h�y Ó�w	��r*-�0�������Bf��J&	u:fIV�r�:rjh.p�甴�9�M��H���>���듲pg�BҲ4:{-��RT�P�MHZ��Rw+|�)�D�qKєW2W��e����j�O{F|��aM)xh���ʴH��-�IF�3捩Gِ\0\0\0\0\0�q���d-�B\\*�u���D�Z��nǮ~�4��gfG�y�N��-m�ݯ�m(ָ����m����WW���IoԺ������w2I��HX%�Lza�֞X��Y��;�$�D�Q{}�54;9�)k3S���2%-|Ĕ����\0��E�ɧT|+2�B��B�H�U�y�������K>Yk�[\ZYN��\"ZyTGV\ZĢ5MGR�OO��&7����;_��Ou��[7���O��?�0�Rv�Vg#۔��+���E�1�[ٲ\0\0\0\0\0��;X��\Zqz��qJ����2c:�#��vIK^Fk4�B/�X���﹑�E4�eh�M8�8(�:�\0<b[�M�=eMB3�\0	������\0\0h���4xrkꬓ��\"�Ƥ�O�?g����_×�����Q�S���	o�9�3�ˉ��&[w���g�n�<�VF�bf�����N�:�ɠ��Z*\0\0\0\0\0����\\��.�۷��C���6���yC��2t2$��3�K�����X~B�7��tZ_`j�8��?&n2.�x��Iڙ�����,�Ø�f6�1{�7�����=snd�;�|e��en��T�Ri-�$���2\"/�\0\0��j�\\�:Z�0�\rNԨfe8��٭��Lz����$�A%\n�T��\\	\\�[*l��;\ZJ�jt�UeJ�������ק���a{1�1���\Z�.�VdϚ����Y�#�������J�p\'\"����gw���&�T��R�Xڌ��\0\0\0\0\0��c�{0q0оS��h�����<���7h�����]I��UJ���D����:�g�{]BA�(���8�i����m��]b��;�bez�]��n@���.�.�Qs�{�G���2؍$j]�]��6ꏇ{&�$��+e�^��X]ܤ����%�>s2ia���
```

Comment:
The data isn't looking so fine. Since here there shouldn't be any changes, could you replace this single file with the unchanged version?